hummingbot/connector/exchange/kraken/kraken_utils.py

```diff
@@ -108,4 +194,13 @@ def is_dark_pool(trading_pair_details: Dict[str, Any]):
                   required_if=using_exchange("kraken"),
                   is_secure=True,
                   is_connect_key=True),
+    "kraken_api_tier":
+        ConfigVar(key="kraken_api_tier",
+                  prompt="Enter your Kraken API Tier(i.e Starter/Intermediate/Pro) >>> ",
+                  required_if=using_exchange("kraken"),
+                  default="starter",
+                  is_secure=False,
+                  is_connect_key=True,
+                  validator=lambda v: _api_tier_validator(v),
+                  ),
```

Comment:
It might be confusing for the user that the values provided in the prompt as clarification are in title case, but the default value is all lowercase